component/otelcol/exporter/loki/internal/convert/convert.go

```diff
@@ -92,7 +93,7 @@ func (conv *Converter) ConsumeLogs(ctx context.Context, ld plog.Logs) error {
 				removeAttributes(log.Attributes(), mergedLabels)
 				removeAttributes(resource.Attributes(), mergedLabels)
 
-				entry, err := convertLogToLokiEntry(log, resource, format)
+				entry, err := convertLogToLokiEntry(log, resource, format, scope)
```

Comment:
Could we use the `LogToLokiEntry` from the upstream translator package directly her from line 72 onwards (on the innermost loop)?

We then can remove the extra code (if you prefer, you can leave a TODO comment and I'll do the cleanup afterwards)

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0ae07ba41d8e19bf684ed3b793060299c4a2083d/pkg/translator/loki/logs_to_loki.go#L133